package.json

```diff
@@ -57,7 +57,7 @@
     "node-uuid": "^1.4.3",
     "nodemailer": "^2.3.2",
     "nomnom": "^1.8.1",
-    "rdflib": "^0.8.0",
+    "rdflib": "git://github.com/linkeddata/rdflib.js#dz_match_alias",
```

Comment:
So that bit is just to give us extra time to review this PR, while the rdflib PR https://github.com/linkeddata/rdflib.js/pull/141 is being reviewed/merged.
I'll update it with the actual published rdflib version before merging this one.